src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -1109,7 +1109,7 @@ paths:
         '201':
           description: Successful response
           schema:
-            $ref: '#/definitions/CategoryEntry'
+            $ref: '#/definitions/CategoryPaging'
```

Comment:
`CategoryEntry` is the correct ref as it is consistent with other APIs such as create nodes. As you can see from the description:

> If you specify a list as input, then a paginated list rather than an entry is returned in the response body

Reply:
At one point I totally agree - we should be consistent with other endpoint definitions and revert that change as it is the only one that stands out (however, it does not show false information).
On the other hand - our Web/REST framework is completely not compatible with Open API definitions ver. 2 which allows only one object per response status. 
As you know, our Web/REST framework returns either an entry or a list of entries with pagination data on successful status.

Reply:
If this is a convention than I will revert to `CategoryEntry`.

Reply:
Yes, we are not fully compatible with OpenAPI Spec (these was a conscious decision that we made before). So consistency is the key here. This is important for the user experience of our API.